content/index.html

```diff
@@ -357,6 +333,13 @@ <h4 class="title">Latest Release
                                         <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/Apache+CarbonData+1.3.1+Release"
                                            class="whatsnew" target="_blank">what's new</a>
                                     </div>
+                                    <div class="block-row">
+                                        <a href="https://dist.apache.org/repos/dist/release/carbondata/1.3.0/"
+                                           target="_blank">Apache CarbonData 1.3.0</a>
+                                        <span class="release-date">Feb 2018</span>
+                                        <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/Apache+CarbonData+1.3.0+Release"
+                                           class="whatsnew" target="_blank">what's new</a>
+                                    </div>
                             </div>
                             <div class="nextR">
```

Comment:
Could not find any content related to CarbonData Release notes 0.2.0 in he index.html published. 

Can this content be removed/